modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicCogOnlineTest.java

```diff
@@ -247,19 +254,18 @@ public void testEmptyMosaic() throws Exception {
     /** Checking time get extracted from remote URL too. */
     @Test
     public void testTimeDimensionMosaic() throws Exception {
-
         final File workDir = prepareWorkingDir("emptycog.zip", "timeMosaic", "");
         try (FileWriter out =
                 new FileWriter(
                         new File(TestData.file(this, "."), "/timeMosaic/datastore.properties"))) {
-            out.write("database=cogmosaic\n");
+            out.write("database=cogtimemosaic\n");
```

Comment:
Is it going to create the db during the test?

See also the setup of the online test:
https://github.com/geotools/geotools/blob/main/.github/workflows/postgis_online.yml

Reply:
Yep. it's just H2 db